types/discord.ts

```diff
@@ -707,7 +706,7 @@ export interface DiscordChannel {
   position?: number;
   /** The name of the channel (1-100 characters) */
   name?: string;
-  /** The channel topic (0-1024 characters) */
+  /** The channel topic (0-4096 characters for GUILD_FORUM channels, 0-1024 characters for all others) */
```

Comment:
Does this need updating in the permission plugin, incase we were throwing errors for anything above 1024 before